### PR TITLE
bug fixes across meshery docs

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR fixes #
1. Alignment of tables description across api reference tables
image
there are more tables like this
2. Dark mode bug fixed
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 6f9ff498abeea463891db657540b59ac7d3fbb25 at: https://6423ebd06dac136b81811684--meshery-docs-preview.netlify.app

**Comment by sudhanshutech** [comment]
image

**Comment by l5io** [comment]
🚀 Preview for commit 57da26e65bdca833df329325dada04bb5537e5d5 at: https://642400cad369af7ee5969421--meshery-docs-preview.netlify.app

File: docs/_sass/table.scss [hunk 1]
```diff
@@ -493,3 +493,8 @@ table.tablesorter th.headerSortUp {
 td.details, .hidden-details > td {
   background-color: var(--color-primary-dark) !important;
 }
+
+
+#compatibility-table  {
+  text-align: center;
```
Comment (leecalcote) [inline docs/_sass/table.scss:499]: Not desired. Left-aligned content is preferred.

File: docs/_sass/table.scss [hunk 2]
```diff
@@ -493,3 +493,8 @@ table.tablesorter th.headerSortUp {
...
+}
```
Comment (leecalcote) [inline docs/_sass/table.scss:500]:
```suggestion
```

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Review by leecalcote — APPROVED** [review]

**Pull request merged**